src/models/multiChart.js

```diff
@@ -213,6 +213,9 @@ nv.models.multiChart = function() {
                 return a.map(function(aVal,i){return {x: aVal.x, y: aVal.y + b[i].y}})
             }).concat([{x:0, y:0}]) : [];
 
+            x = lines1.xScale();
```

Comment:
"add a new option on multiChart named xScale and have it then control the xScale method on all subcharts"

YES! THIS IS EXACTLY RIGHT!

Sorry I've been tearing my hair out trying to get this work to allow for multilingual support and the D3 timeFormat.multi capability ><